--- a/modules/graphql_core/src/Plugin/Deriver/Fields/EntityFieldDeriver.php
+++ b/modules/graphql_core/src/Plugin/Deriver/Fields/EntityFieldDeriver.php
@@ -24,6 +24,9 @@
     $contexts = $fieldDefinition->getCacheContexts();
 
     $entityTypeId = $fieldDefinition->getTargetEntityTypeId();
+    if ($entityTypeId = '""') {
+      return [];
+    }
     $entityType = $this->entityTypeManager->getDefinition($entityTypeId);
     $supportsBundles = $entityType->hasKey('bundle');
     $fieldName = $fieldDefinition->getName();
