--- a/modules/helfi_etusivu_entities/src/Plugin/ExternalEntities/StorageClient/Surveys.php
+++ b/modules/helfi_etusivu_entities/src/Plugin/ExternalEntities/StorageClient/Surveys.php
@@ -32,8 +32,9 @@ final class Surveys extends EtusivuJsonApiEntityBase {
   public function query(
     array $parameters = [],
     array $sorts = [],
-    $start = NULL,
-    $length = NULL,
+    ?int $start = NULL,
+    ?int $length = NULL,
+    array &$unhandled_filters = [],
   ) : array {
     $query = [
       'fields[node--survey]' => 'id,langcode,status,published_at,unpublish_on,title,body,field_survey_link',

--- a/modules/helfi_etusivu_entities/src/Plugin/ExternalEntities/StorageClient/Announcements.php
+++ b/modules/helfi_etusivu_entities/src/Plugin/ExternalEntities/StorageClient/Announcements.php
@@ -32,8 +32,9 @@ final class Announcements extends EtusivuJsonApiEntityBase {
   public function query(
     array $parameters = [],
     array $sorts = [],
-    $start = NULL,
-    $length = NULL,
+    ?int $start = NULL,
+    ?int $length = NULL,
+    array &$unhandled_filters = [],
   ) : array {
     $query = [
       'fields[node--announcements]' => 'id',

--- a/modules/helfi_paragraphs_hearings/src/Plugin/ExternalEntities/StorageClient/Hearings.php
+++ b/modules/helfi_paragraphs_hearings/src/Plugin/ExternalEntities/StorageClient/Hearings.php
@@ -86,8 +86,9 @@ class Hearings extends StorageClientBase {
   public function query(
     array $parameters = [],
     array $sorts = [],
-    $start = NULL,
-    $length = NULL,
+    ?int $start = NULL,
+    ?int $length = NULL,
+    array &$unhandled_filters = [],
   ) : array {

     $langcode = $this->languageManager

--- a/modules/helfi_paragraphs_curated_event_list/src/Plugin/ExternalEntities/StorageClient/Events.php
+++ b/modules/helfi_paragraphs_curated_event_list/src/Plugin/ExternalEntities/StorageClient/Events.php
@@ -81,8 +81,9 @@ class Events extends RestClient {
   public function query(
     array $parameters = [],
     array $sorts = [],
-    $start = NULL,
-    $length = NULL,
+    ?int $start = NULL,
+    ?int $length = NULL,
+    array &$unhandled_filters = [],
   ) : array {
     $langcode = $this->languageManager

--- a/modules/helfi_paragraphs_news_list/src/ElasticExternalEntityBase.php
+++ b/modules/helfi_paragraphs_news_list/src/ElasticExternalEntityBase.php
@@ -277,8 +277,9 @@
   public function query(
     array $parameters = [],
     array $sorts = [],
-    $start = NULL,
-    $length = NULL,
+    ?int $start = NULL,
+    ?int $length = NULL,
+    array &$unhandled_filters = [],
   ) : array {
     $query = $this->buildQuery($parameters, $sorts);
